### PR TITLE
feat(terraformer): added terraformer integration

### PR DESCRIPTION
**Pull request by joaoReigota1**
**Proposed Changes**
- added terraformer integration with kics

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@ nav:
       - AWS CDK: cdk.md
       - Badge: BADGE.md
       - Using pre-commit hooks: integrations_pre_commit.md
+      - Terraformer: integrations_terraformer.md
```
Comment (rafaela-soares) [inline mkdocs.yml:73]: Also added it in `docs/integrations.md`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
LGTM 🚀 🚀 🚀 
Only a minor suggestion:

File: docs/integrations_terraformer.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
+# Running KICS with Terraformer for AWS
+
```
Comment (nunoocx) [inline docs/integrations_terraformer.md:2]: Add an intro like so:

From version 1.5, KICS integrates with Terraformer to scan resources deployed in the cloud. The runtime information of the resources is obtained by providing a Terraformer path to KICS, via `-p` flag. The scan happens immediately after this information is obtained. In the end, results are shown as for any other KICS scan.

#### Cloud providers supported:
- AWS

**Review by nunoocx — CHANGES_REQUESTED** [review]

**Review by nunoocx — APPROVED** [review]
LGTM

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Pull request merged**